src/radical/entk/appman/wfprocessor.py

```diff
@@ -73,49 +77,54 @@ def __init__(self,
         self._logger.info('Created WFProcessor object: %s' % self._uid)
         self._prof.prof('wfp obj created', uid=self._uid)
 
-    # ------------------------------------------------------------------------------------------------------------------
+
+    # --------------------------------------------------------------------------
+    #
     # Getter
-    # ------------------------------------------------------------------------------------------------------------------
+    #
     @property
     def workflow(self):
         return self._workflow
 
-    # ------------------------------------------------------------------------------------------------------------------
-    # Private Methods
-    # ------------------------------------------------------------------------------------------------------------------
 
+    # --------------------------------------------------------------------------
+    #
+    # Private Methods
+    #
     def _initialize_workflow(self):
         """
-        **Purpose**: Initialize the PST of the workflow with a uid and type checks
+        **Purpose**: Initialize the PST of the workflow with uid and type checks
         """
 
         try:
-
             self._prof.prof('initializing workflow', uid=self._uid)
 
             for p in self._workflow:
                 p._assign_uid(self._sid)
 
             self._prof.prof('workflow initialized', uid=self._uid)
 
-        except Exception, ex:
-            self._logger.exception(
-                'Fatal error while initializing workflow: %s' % ex)
+        except Exception:
```

Comment:
because it's re-raised and logged again.  I don't think it's useful to do the `try: ... except: raise` in the first place, really: just letting the exception fall through will tell you the same thing...